R/cassette_class.R

```diff
@@ -441,32 +422,13 @@ Cassette <- R6::R6Class(
       self$record == "all"
     },
 
-    #' @description Get character string of entire cassette; bytes is a misnomer
-    #' @return character
-    raw_cassette_bytes = function() {
```

Comment:
Only used by `is_empty()` which now just looks to see if the file exists

Reply:
yep, good move